### PR TITLE
Added extension api datasets

### PR DESCRIPTION
**Pull request by JSalphanauten**
### 1. Why is this change necessary?
To make more datasets available to the admin extension sdk.

### 2. What does this change do, exactly?
Used the code snippet provided by shopware(product detail) as template to make datasets available for several detail pages.

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-cms/page/sw-cms-detail/index.js [hunk 1]
```diff
@@ -325,6 +325,11 @@ Component.register('sw-cms-detail', {
 
     methods: {
         createdComponent() {
+            Shopware.ExtensionAPI.publishData({
+                id: 'sw-cms-detail__cms',
```
Comment (seggewiss) [inline src/Administration/Resources/app/administration/src/module/sw-cms/page/sw-cms-detail/index.js]:
```suggestion
                id: 'sw-cms-detail__page',
```

**Review by seggewiss — CHANGES_REQUESTED** [review]
Looking really good so far 👍

**Review by nfech01 — APPROVED** [review]
Page is the correct naming.

**Comment by jleifeld** [comment]
Can you have a look at the Jest tests? There are some failing tests which need to be fixed before we can merge it

**Comment by JSalphanauten** [comment]
> Can you have a look at the Jest tests? There are some failing tests which need to be fixed before we can merge it

@jleifeld which test exactly are you talking about?
I haven't noticed dependencies to any test.

**Comment by jleifeld** [comment]
> > Can you have a look at the Jest tests? There are some failing tests which need to be fixed before we can merge it
> 
> @jleifeld which test exactly are you talking about? I haven't noticed dependencies to any test.

You can execute the admin jest tests by following composer command:
`composer run admin:unit` or (for the watcher) `composer run admin:unit:watch`

Then you see the failing tests which needs to be fixed or updated

(https://developer.shopware.com/docs/guides/plugins/plugins/testing/jest-admin)

**Comment by jleifeld** [comment]
I updated the tests and merged it with https://github.com/shopware/shopware/commit/d7bda7ede5c76d4ce5e6b99850df334d6eb4a109 to the platform.

Thanks for your contribution!

**Pull request closed without merging**